_posts/2019-04-17-sod.md

```diff
@@ -0,0 +1,58 @@
+
+## Project Ideas List
+
+### Project 1: Improve Documentation of Stream Processing Concepts
+
+**Description:** Stream processing is the processing of data in motion―in other words, computing on data directly as it is produced or received. Apache Flink has pioneered the field of distributed, stateful stream processing over the last several years. As the community has pushed the boundaries of stream processing, we have introduced new concepts that users need to become familiar with to develop and operate Apache Flink applications efficiently.
+The Apache Flink documentation \[1\] already contains a “concepts” section, but it is a ) incomplete and b) lacks an overall structure & reading flow. In addition, “concepts”-content is also spread over the development \[2\] & operations \[3\] documentation without references to the “concepts” section. An example of  this can be found in \[4\] and \[5\].
+
+In this project, we would like to restructure, consolidate and extend the concepts documentation for Apache Flink to better guide users who want to become productive as quickly as possible. This includes better conceptual introductions to topics such as event time, state, and fault tolerance with proper linking to and from relevant deployment and development guides.
+
+**Related material:**
+
+1. https://ci.apache.org/projects/flink/flink-docs-release-1.8/
+2. https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev
+3. https://ci.apache.org/projects/flink/flink-docs-release-1.8/ops/
+4. https://ci.apache.org/projects/flink/flink-docs-release-1.8/concepts/programming-model.html#time
+5. https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/event_time.html
+
+### Project 2: Improve Documentation of Flink Deployments & Operations?
+
+**Description:** Stream processing is the processing of data in motion―in other words, computing on data directly as it is produced or received. Apache Flink has pioneered the field of distributed, stateful stream processing for the last few years. As a stateful distributed system in general and a continuously running, low-latency system in particular, Apache Flink deployments are non-trivial to setup and manage.
+Unfortunately, the operations \[1\] and monitoring documentation \[2\] are arguably the weakest spots of the Apache Flink documentation. While it is comprehensive and often goes into a lot of detail, it lacks an overall structure and does not address common overarching concerns of operations teams in an efficient way.
+
+In this project, we would like to restructure this part of the documentation and extend it if possible. Ideas for extension include: discussion of session and per-job clusters, better documentation for containerized deployments (incl. K8s), capacity planning &  integration into CI/CD pipelines.
```

Comment:
`Ideas for extension include: discussion of session and per-job clusters,` -> `Ideas for the extension included: discussion of the session and per-job clusters,`?

Reply:
Not sure. I'll leave it as is, as Seth (native speaker) did not complain ;)